docs/build_and_load.md

```diff
@@ -327,10 +327,10 @@ goals:
 1. Copy and rename FASTA files via user-provided sample name IDs.
 2. Add a sample name tag to the ID lines of each FASTA file.
 
-> [!WARNING]
-> **These updated assembly FASTA files need to be used as input for 
-> both `agc-compress` and `align-assemblies` steps.** This ensures 
-> consistent sample names across the pipeline!
+!!! warning
+    **These updated assembly FASTA files need to be used as input for 
+    both `agc-compress` and `align-assemblies` steps.** This ensures 
+    consistent sample names across the pipeline!
 
```

Comment:
While agc-compress and align-assemblies are the most important commands that need the updated fastas, users should also use the updated reference fasta files as parameters to the other commands that take a reference, e.g. CreateMafVcf, CreateRanges, CreatRefVcf.  Can we leave what you have, but include a general statement that all PHG commands that take a fasta file should be given the updated reference or assembly FASTA files as the file parameter?  

I want to be sure users understand the reference fasta also needs to be updated. (at least when stored in AGC)

Reply:
I have updated the warning block to include the some additional text you have brought up.